### ISSUE TITLE
TMC2130 & MAX6675 problem

### ISSUE THREAD

**Issue by cubical23** [repetier/Repetier-Firmware#803]
Hi all,

I'm currently building a new printer and I'm trying to use TMC2130 and MAX6675 at the same time (connected to a RADDS 1.5 board, it's 4 steppers and a single MAX6675 at the moment).

But there seems to be a problem: when enabling TMC2130 support in firmware and enabling MAX6675 too, the SD-card and the readings from MAX6675 are not working anymore (temperature reading is always 0, like not connected, SD-card insert/removal is detected but files are not accessable). 

If I disable either TMC2130 support or disable MAX6675 support, SD-card is working. Also when disabling TMC2130 support, MAX6675 readings are fine. 

Any suggestions? I guess it might be some SPI problem. 

Thanks in advance

**Comment by repetier** [comment]
Yes I also would think it is SPI. I do not have the hardware for tests but guess the MAX6675 does not like mode settings from sd card. One of the reasons I want to switch the the arduino spi library in V2.

**Comment by xarwell** [comment]
Hi,

I also have the same problem. @repetier can you term date of next update ?

**Comment by repetier** [comment]
You mean V2? Not really as it depends on too many factors. It already works for some simple configurations, but it will take time and also other users to make it final. Especially on hardware that I do not own my self.

I hope to have good basic implementation and some special drivers by the end of the year so that at least all features are implemented. With the new V2 we have a more modular system, so modifications should become much easier, so my hope is to also get more user contributions with new device supports.

**Comment by xarwell** [comment]
Ok, thanks.
